src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -67,7 +67,7 @@ def _splitname_parser(line):
 
     :returns: dictionary with stripped prefixes
     """
-    parts = line[0].split(';')
+    parts = line.split(';')
```

Comment:
now this is where I would try to look for an empty line and complain about it, returning a `{ }`